common/logger/logger.ts

```diff
@@ -10,7 +10,10 @@ addLayout('json', function () {
         const error = logEvent.context['error'] || {};
         delete logEvent.context['error'];
 
-        return JSON.stringify({ ...logEvent, message, data, error });
+        // These tags will be used to identify the logs in datadog later on
+        const tags = { env: process.env.ENV, version: process.env.VERSION || 'latest' };
```

Comment:
@Jonasdoubleyou Is it possible to set a VERSION environment variable in Heroku (or maybe there is one already)? 
Maybe something around the commit hash or similar, so that we can distinguish logs of different deployments.

Reply:
Yes, there is now HEROKU_SLUG_COMMIT (c.f. [Dyno Metadata](https://devcenter.heroku.com/articles/dyno-metadata)). Only turned on in dev for now.